src/Spryker/Zed/EventBehavior/Persistence/EventBehaviorQueryContainer.php

```diff
@@ -65,6 +65,22 @@ public function queryLatestEntityChange(DateTime $date)
         return $query;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param int[] $keys
+     *
+     * @return \Orm\Zed\EventBehavior\Persistence\SpyEventBehaviorEntityChangeQuery
+     */
+    public function queryEntityByKeys(array $keys): SpyEventBehaviorEntityChangeQuery
```

Comment:
First of all method's name is a bit wrong. `Keys` is generic. It's not obvious that we are talking about primary keys.
Also this might be a good time to introduce repository in this module.

Reply:
I removed this method and added Entity manager with the method deleteEventBehaviorEntityByPrimaryKeysIds, please review